theia-extensions/theia-blueprint-updater/src/electron-main/update/theia-updater-impl.ts

```diff
@@ -62,7 +62,8 @@ export class TheiaUpdaterImpl implements TheiaUpdater, ElectronMainApplicationCo
         });
 
         autoUpdater.on('error', (err: unknown) => {
-            this.clients.forEach(c => c.reportError('Error during update'));
+            const errorLogPath = autoUpdater.logger.transports.file.getFile().path;
+            this.clients.forEach(c => c.reportError({ message: 'Error during update', errorLogPath }));
```

Comment:
We might want to improve the `message` while we are modifying the code. I'm open to any suggestions you might have, or even keeping it as is:

```
message: 'An error has occurred while attempting to update.'
```